package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "web-tooling-benchmark",
   "description": "JavaScript Benchmark for common Web Developer workloads",
-  "version": "0.3.2",
+  "version": "0.4.0",
```

Comment:
Please don’t bump the project’s version number. That needs to be done by maintainers in a separate commit.

Reply:
Ah, okey, sorry @mathiasbynens. I read it here https://github.com/v8/web-tooling-benchmark/issues/31#issuecomment-357578201

Reply:
https://github.com/v8/web-tooling-benchmark/issues/31#issuecomment-357596714 came later and was then agreed upon. But hey, no worries!

Thanks for contributing this patch and linking to the prettier and source-map issues!